addons/l10n_hu_edi/models/res_company.py

```diff
@@ -0,0 +1,238 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, _
+from odoo.exceptions import UserError
+from odoo.addons.l10n_hu_edi.models.l10n_hu_edi_connection import L10nHuEdiConnection, L10nHuEdiConnectionError, XML_NAMESPACES
+
+from lxml import etree
+import base64
+from datetime import timedelta
+
+
+L10N_HU_EDI_SERVER_MODE_SELECTION = [
+    ('production', 'Production'),
+    ('test', 'Test'),
+]
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    l10n_hu_group_vat = fields.Char(
+        related='partner_id.l10n_hu_group_vat',
+        readonly=False,
+    )
+    l10n_hu_tax_regime = fields.Selection(
+        selection=[
+            ('ie', 'Individual Exemption'),
+            ('ca', 'Cash Accounting'),
+            ('sb', 'Small Business'),
+        ],
+        string='Hungarian Tax Regime',
+    )
+    l10n_hu_edi_server_mode = fields.Selection(
+        selection=L10N_HU_EDI_SERVER_MODE_SELECTION,
+        string='Server Mode',
+    )
+    l10n_hu_edi_username = fields.Char(
+        string='NAV Username',
+        groups='base.group_system',
+    )
+    l10n_hu_edi_password = fields.Char(
+        string='NAV Password',
+        groups='base.group_system',
+    )
+    l10n_hu_edi_signature_key = fields.Char(
+        string='NAV Signature Key',
+        groups='base.group_system',
+    )
+    l10n_hu_edi_replacement_key = fields.Char(
+        string='NAV Replacement Key',
+        groups='base.group_system',
+    )
+    l10n_hu_edi_last_transaction_recovery = fields.Datetime(
+        string='Last transaction recovery (in production mode)',
+        default=lambda self: fields.Datetime.now(),
+    )
+
+    def _l10n_hu_edi_configure_company(self):
+        """ Single-time configuration for companies, to be applied when l10n_hu_edi is installed
+        or a new company is created.
+        """
+        for company in self:
+            # Set profit/loss accounts on cash rounding method
+            profit_account = self.env['account.chart.template'].with_company(company).ref('l10n_hu_969', raise_if_not_found=False)
+            loss_account = self.env['account.chart.template'].with_company(company).ref('l10n_hu_869', raise_if_not_found=False)
+            rounding_method = self.env.ref('l10n_hu_edi.cash_rounding_1_huf', raise_if_not_found=False)
+            if profit_account and loss_account and rounding_method:
+                rounding_method.with_company(company).write({
+                    'profit_account_id': profit_account.id,
+                    'loss_account_id': loss_account.id,
+                })
+
+            # Activate cash rounding on the company
+            res_config_id = self.env['res.config.settings'].create({
+                'company_id': company.id,
+                'group_cash_rounding': True,
+            })
+            res_config_id.execute()
```

Comment:
No, because the cash rounding is applied by adding the user to the group.
See l10n_ch for another example of this: https://github.com/odoo/odoo/blob/a93472a4b21fd1349691ecbcfdfea2b75b34589b/addons/l10n_ch/__init__.py#L16